docs/markdown/snippets/install-many-emptydirs.md

```diff
@@ -0,0 +1,5 @@
+## [[install_emptydir]] now supports multiple dirs inside one call
+
+Prior to this release passing multiple paths as `dirpath` argument was resulting in only the first path to get created. The rest were ignored. That was expected behavior as [[install_emptydir]] was documented to only take a single path. But it also had an incorrect type annotation that claimed for it to take multiple paths, which led to confusion.
```

Comment:
Again, I'm just following the stile. If you open [release notes for v1.3.0](https://mesonbuild.com/Release-notes-for-1-3-0.html) you'll similarly find that the first section "Clarify of implicitly-included headers in C-like compiler checks" explains also the reasoning behind the change. Is that bad?

Reply:
The (large) difference is that the release notes for implicitly included headers in C-like compiler checks because they are important background information for users to know why the compiler checks now behave differently when used, and also for users to know how to adapt to that if needed.

Here, the "reasoning behind the change" is information about meson's type annotations.

If you're discussing the mypy-checked PEP 484 type hints in the release notes for a build system, you're doing "it" wrong.

But also, this release notes snippet just has too much information to sift through in order for users to find out what they care about. With the other snippet, users *have* to know about the change to stdio.h in order to know *what they should do*.

Reply:
Done! I removed almost all explanation and only left

> Prior to this release passing multiple paths as `dirpath` argument was resulting in only the first path to get created. Now passing more than one path in `dirpath` will create all of them.